### PR TITLE
some stuff i know/worked with

### PR DESCRIPTION
**Pull request by DrAzraelTod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +245,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [NewsBlur](https://github.com/samuelclay/NewsBlur) - NewsBlur is a personal news reader that brings people together to talk about the world. A new sound of an old instrument
   * [Reader-Self](http://readerself.com/) Self-hosted rss reader (php / mysql or sqlite) - Google Reader alternative ([Source code](https://github.com/readerself/readerself))
   * [Readr](https://github.com/pabloprieto/Readr) - A clean & simple, self-hosted RSS reader
+  * [RSS2EMail](http://www.allthingsrss.com/rss2email/) - exactly what you'd expect from it's name `GPL` `Python`
```
Comment (nodiscc) [inline README.md]: Please a relevant description; "tool for getting news from RSS feeds in email" seems ok.

**Comment by Kickball** [comment]
Once you've done these small things we'll be able to merge:

* Add a short description (<250 characters) for Webalizer
* Add a short description (<250 characters) for WebChao
* Podcatcher.sh uses multiple languages, either list the primary language, or use a / to separate them. Example: `Bash/JS`

File: README.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ Table of Contents
   * [Open Web Analytics](http://www.openwebanalytics.com/) - Google Analytics and Piwik alternative - `GPL` `PHP`
   * [Piwik](https://piwik.org/) - Leading open-source analytics platform that gives you more than just powerful analytics. - `GPLv3` `PHP`
   * [ThinkUp](https://www.thinkup.com/) - Insights into your social networking activity on Twitter, Facebook, Instagram, and beyond. ([Source code](https://github.com/ThinkUpLLC/ThinkUp)) - `GPL` `PHP`
+  * [Webalizer](http://www.webalizer.org/) - `GPL` `C`
```
Comment (nodiscc) [inline README.md]: Please add a description; "fast, free web server log file analysis program" would be ok.

File: README.md [hunk 3]
```diff
@@ -89,6 +90,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Known](https://withknown.com/opensource/) - A single website for all your content `Apache` `PHP`
   * [Noddity](http://noddity.com/) - It's a blog, it's a wiki, it's a fast CMS! ([Source code](https://github.com/TehShrike/noddity)) `WTFPL` `Nodejs`
   * [PluXml](http://www.pluxml.org/) - XML-based blog/CMS platform ([Source code](https://github.com/pluxml/PluXml)) `GPL` `PHP`
+  * [WebChao](https://github.com/drazraeltod/webchao) `GPLv3` `Python` `Django`
```
Comment (nodiscc) [inline README.md]: There is no description of what this does in the repo.
Please the language tags to one.

File: README.md [hunk 4]
```diff
@@ -318,6 +321,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Emby](https://github.com/MediaBrowser/MediaBrowser) - A home media server built using other popular open source technologies. - `GPLv2`
   * [Flumotion](http://flumotion.net/) - a streaming media server with intuitive graphical administration tools ([Source code](https://github.com/Flumotion/flumotion)) `LGPLv2` `Python`
   * [HTPC Manager](http://htpc.io/) - A fully responsive interface to manage all your favorite media on your HTPC. - `MIT`
+  * [Podcatcher.sh](https://github.com/DrAzraelTod/podcatcher.sh) - gets Podcasts by wget, builds m3u-Playlist, contains m3u.js that builds html5-Player for Playlists - `GPL` `Bash` `JS`
```
Comment (nodiscc) [inline README.md]: I don't know if this is relevant to this list. These are mostly shell scripts to build an html index/m3u playlist for podcasts.

> Does it work?
> Well.. some of it does

I see this is your own project, we currently have no quality requirements, but I'd prefer if you let it mature a bit/improve it before adding it to the list (not invoking subshells with backticks for example)

Is this ok?

File: README.md [hunk 5]
```diff
@@ -350,9 +354,11 @@ CMS are a practical way to setup a website with many features. CMS often come wi
    * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
 
 
+
 ## Music Streaming
 
   * [Ampache](http://ampache.org/) - A web based audio/video streaming application `GPLv2` `PHP`
+  * [CherryMusic](http://www.fomori.org/cherrymusic/) minimalistic Web-Mediaplayer - `GPL` `Python`
```
Comment (nodiscc) [inline README.md]: missing ` - ` between title and description

**Comment by nodiscc** [comment]
@DrAzraelTod Thanks for the patch. Can you fix the points mentioned above? Everything can be discussed

**Comment by DrAzraelTod** [comment]
i will edit it, later today :-)

**Comment by DrAzraelTod** [comment]
i removed webchao and podcatcher.sh completely
amongst the two podcatcher is probably even the one easier to adopt for someone else, so it wouldn't make any sense to keep them in here.

while thinking further about this, i probably should just post the m3u.js-part of podcatcher (works standalone) and link to [podget](http://podget.sourceforge.net/) (although that's nothing to actually "host") and omit my hackish bash-glue (traverses directories, builds playlists) completely.

but then i'd have neither a page nor anything else i could link to for that, so i'll just keep it as is for now.

**Comment by Kickball** [comment]
Lovely, thanks for the contribution.

**Pull request merged**